samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs

```diff
@@ -34,15 +36,7 @@ public class RazorViewToStringRenderer
         public async Task<string> RenderViewToStringAsync<TModel>(string name, TModel model)
```

Comment:
On second thought, the current name matches what we have in `ViewResult`: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewResult.cs#L30. I'll change it to `viewName` and call it a day (particularly given that it's not necessarily a path all the time).